doc/reports/prelude.rst

```diff
@@ -0,0 +1,82 @@
+
+Zephyr Test Result Report
+############################
+
+
+Overview
+********
+
+This report provides the most up to date information about the health and quality of the current Zephyr RTOS version  . This report gathers key metrics describing the state and performance of the current version and the results of its testing. Whenever applied, references for detailed analysis are provided. This report is a part of the Quality Assurance process for the Zephyr RTOS product.
+
+
+Word format file can be found `here
+<https://docs.google.com/document/d/12kEprn7uBkD238EZmQkvD9_hHV9ttcea8Ke-MszLRnc/edit>`_

```

Comment:
Document by that URL is long outdated. And why mention it here at all if this text version is supposed to be used by people outside of Testing WG?

Reply:
ok, I remove it